.github/workflows/unit_tests_nightly.yml

```diff
@@ -22,7 +22,7 @@ jobs:
           miniforge-variant: Mambaforge
           activate-environment: mslice-env
           environment-file: environment.yml
-          python-version: 3.8
+          python-version: 3.10
```

Comment:
Do we no longer need this as `python = 3.10` is in the environment.yml file?

Reply:
Same for the `deploy_*` workflows

Reply:
Good spot, it should be redundant by now.